test/hotspot/jtreg/compiler/inlining/ResolvedClassTest.java

```diff
@@ -25,7 +25,7 @@
  * @test
  * @bug 8279515
  *
- * @requires vm.flagless
+ * @requires vm.flagless & vm.compMode != "Xint" & vm.compiler1.enabled & vm.compiler2.enabled
```

Comment:
Thanks for the update.

Since there are already `vm.compiler1.enabled & vm.compiler2.enabled` and `vm.flagless`, `vm.compMode != "Xint"` seems to be unnecessary.

So we can safely drop `vm.compMode != "Xint"` even with zero-vm, right?

Reply:
OK， DONE

Reply:
> > /integrate
> 
> Let's wait for two days to see if @dean-long and others have better solution.

Sure. Thank you, jiege